docs/rxd-tutorials/basic-initialization.ipynb

```diff
@@ -2,111 +2,164 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "fFdOnTv7nfzq"
+   },
+   "source": [
+    "A version of this notebook may be run online via Google Colab at https://tinyurl.com/rxd-basic-initialization (make a copy or open in playground mode)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "C93hhSQ4nKqC"
+   },
    "source": [
     "The time series of a chemical concentration necessarily depends on its initial conditions; i.e. the concentration at time 0. An analogous statement is true for gating variables, etc. How do we specify this?"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "n_OCck5BnKqK"
+   },
    "source": [
     "<h2>Option 1: NEURON and NMODL defaults</h2>"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "id": "rBjwbSL7nKqL"
+   },
    "source": [
     "If the species corresponds to one with initial conditions specified by NMODL (or in the case of sodium, potassium, or calcium with meaningful NEURON defaults), then omitting the initial argument will tell NEURON to use those rules. e.g."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
```

Comment:
Is all the metadata needed?

Reply:
No. Is there a non-manual way of cleaning it?

Reply:
I believe I used this when clearing notebooks history: 
https://pypi.org/project/nb-clean/